spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -53,26 +46,83 @@
     end
   end
 
+  describe "show" do
+
+    it "returns an individual skill" do
+      skill = create(:skill, name: "show_skill")
+
+      get :show, format: :json, id: skill.id
+
+      expect(response.status).to eq 200
+      expect(json_response_skill_name).to eq("show_skill")
+    end
+  end
+
+  describe "update" do
+
+    it "updates a skill" do
+      group_1 = create(:group, name: "before_updated_group")
+      group_2 = create(:group, name: "after_updated_group")
+      skill = create(:skill, name: "before_updated_name", group: group_1)
+
+      put :update, format: :json, id: skill.id, skill: {
+                                                         name: "after_updated_name",
+                                                         featured: true,
+                                                         group: group_2.name
+                                                       }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.